test/custom1dtest.jl

```diff
@@ -22,13 +27,15 @@ solver = ValueIterationSolver(max_iterations=10, belres=1e-3, include_Q=true);
 VIPolicy = DiscreteValueIteration.solve(solver, mdp);
 
 # Solve Finite Horizon by Value Iteration
-FHPolicy = FiniteHorizonPOMDPs.solve(mdp);
+FHPolicy = FiniteHorizonPOMDPs.solve(fhsolver, mdp);
```

Comment:
As a usage example, I think it would be better to call `POMDPs.solve` here instead

Reply:
If I understand this correctly, you are saying that instead of `FiniteHorizonPOMDPs.solve(fhsolver, mdp)` I should use `POMDPs.solve(fhsolver, mdp)`. Correct?
I am calling specific packages to call the desired methods to compare them.

Reply:
My point is coming from the point of view of someone who looks to tests to see how to use the package. As you are defining `solve` as a new method of `POMDPs.solve` and you export it, I think it should be fine just to use `solve(fhsolver, mdp)` here. Anyway, my point is that I was confused by seeing `FiniteHorizonPOMDPs` here as I saw elsewhere that you define it as a new method for `POMDPs.solve`. But you can leave this as is, I don't think it's important.